docs/source/normalizing_data/normalization.rst

```diff
@@ -59,140 +60,310 @@ Normalized gene records are constructed by merging known data from all associate
-                "type": "SequenceReference",
-                "refgetAccession": "SQ.F-LrLMe1SRpfUZHkQmvkVKFEGaoDeHul"
-              },
-              "start": 140713327,
-              "end": 140924929
-            }
-          ]
-        },
-        {
-          "type": "Extension",
-          "name": "strand",
-          "value": "-"
-        },
-        {
-          "type": "Extension",
-          "name": "hgnc_locus_type",
-          "value": "gene with protein product"
-        },
-        {
-          "type": "Extension",
-          "name": "ncbi_gene_type",
-          "value": "protein-coding"
-        },
-        {
-          "type": "Extension",
-          "name": "ensembl_biotype",
-          "value": "protein_coding"
-        }
-      ],
-      "description": null,
-      "xrefs": [
-        "ensembl:ENSG00000157764",
-        "ncbigene:673"
-      ],
-      "alternate_labels": [
-        "B-raf",
-        "BRAF1",
-        "NS7",
-        "B-RAF1",
-        "BRAF-1",
-        "RAFB1"
-      ],
-      "gene": "hgnc:1097"
-    }
+   {
+     "id": "normalize.gene.hgnc:1097",
+     "label": "BRAF",
+     "description": null,
+     "extensions": [
+       {
+         "type": "Extension",
+         "name": "symbol_status",
+         "value": "approved"
+       },
+       {
+         "type": "Extension",
+         "name": "approved_name",
+         "value": "B-Raf proto-oncogene, serine/threonine kinase"
+       },
+       {
+         "type": "Extension",
+         "name": "strand",
+         "value": "-"
+       },
+       {
+         "type": "Extension",
+         "name": "ensembl_locations",
+         "value": [
+           {
+             "id": "ga4gh:SL.WJ0hsPzXuK54mQyVysTqUNV5jaCATnRf",
+             "type": "SequenceLocation",
+             "sequenceReference": {
+               "type": "SequenceReference",
+               "refgetAccession": "SQ.F-LrLMe1SRpfUZHkQmvkVKFEGaoDeHul"
+             },
+             "start": 140719326,
+             "end": 140924929
+           }
+         ]
+       },
+       {
+         "type": "Extension",
+         "name": "ncbi_locations",
+         "value": [
+           {
+             "id": "ga4gh:SL.uNBZoxhjhohl24VlIut-JxPJAGfJ7EQE",
+             "type": "SequenceLocation",
+             "sequenceReference": {
+               "type": "SequenceReference",
+               "refgetAccession": "SQ.F-LrLMe1SRpfUZHkQmvkVKFEGaoDeHul"
+             },
+             "start": 140713327,
+             "end": 140924929
+           }
+         ]
+       },
+       {
+         "type": "Extension",
+         "name": "hgnc_locus_type",
+         "value": "gene with protein product"
+       },
+       {
+         "type": "Extension",
+         "name": "ncbi_gene_type",
+         "value": "protein-coding"
+       },
+       {
+         "type": "Extension",
+         "name": "ensembl_biotype",
+         "value": "protein_coding"
+       }
+     ],
+     "mappings": [
+       {
+         "id": null,
```

Comment:
Exclude nulls?

Reply:
Yeah, is there a reason we don't do this by default?

Reply:
I think I forgot to do the response_model_exclude_none in gene, but have done in other places